.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+This repo is not closely monitored by product and development teams.
+If you are experiencing a product issue, please open the issue on one of these repositories instead.
+
+Common ASP.NET repos and projects:
+
+ - See the full list at https://github.com/aspnet
+
+ - Documentation: https://github.com/aspnet/Docs
+ - MVC: https://github.com/aspnet/Mvc
+ - Entity Framework Core: https://github.com/aspnet/EntityFramework
+ - SignalR: https://github.com/aspnet/SignalR
+ - ASP.NET Core in Docker: https://github.com/aspnet/aspnet-docker
+ - Microsoft.NET.Sdk.Web: https://github.com/aspnet/websdk
+ - Kestrel HTTP Server: https://github.com/aspnet/KestrelHttpServer
+
+Common projects that are not ASP.NET:
+
+ - Visual Studio: https://developercommunity.visualstudio.com
+ - NuGet: https://github.com/NuGet/home
+ - .NET Core CLI, SDK, or runtime: https://github.com/dotnet/core
```

Comment:
dotnet/core is the .NET Core "Home" repo. It contains a list of relevant repos including dotnet/cli, dotnet/sdk, and others.